hibernate-core/src/test/java/org/hibernate/internal/util/xml/XMLHelper.java

```diff
@@ -16,10 +16,11 @@
 /**
  * Small helper class that lazily loads DOM and SAX reader and keep them for fast use afterwards.
  *
- * @deprecated Currently only used for integration with HCANN.  The rest of Hibernate uses StAX now
- * for XML processing.  See {@link org.hibernate.boot.jaxb.internal.stax}
+ * This was part of Hibernate ORM core, but moved into the testsuite helpers to not expose
+ * access to the dom4j types. It's also used by Hibernate Envers, so we will need two copies
+ * until Envers is able to remove its reliance on dom4j.
+ * The rest of Hibernate uses StAX now for XML processing.  See {@link org.hibernate.boot.jaxb.internal.stax}
  */
-@Deprecated
 public final class XMLHelper {
```

Comment:
@Sanne, I'm surprised to see `@Deprecated` removed.

Shouldn't this class be deleted?

Reply:
I moved it into the tests folder: it is removed from the project, but still used in some tests.

Reply:
Oh, ok; I didn't realize.